payjoin/src/weight.rs

```diff
@@ -190,18 +31,10 @@ pub(crate) fn witness_weight(witness: &Witness) -> Weight {
     Weight::from_witness_data_size(size)
 }
 
-impl ComputeSize for Script {
+impl ComputeSize for ScriptBuf {
```

Comment:
I think this should either remain `Script` or have both. Anyway, also something to upstream.